packages/integration-tests/src/EntityManager.test.ts

```diff
@@ -9,6 +9,7 @@ import {
 import { Loaded, setDefaultEntityLimit, setEntityLimit } from "joist-orm";
 import { Author, Book, Publisher, PublisherSize } from "./entities";
 import { knex, newEntityManager, numberOfQueries, queries, resetQueryCount } from "./setupDbTests";
+import {assertNever} from "joist-codegen/build/utils";
```

Comment:
Hm, is there already an `assertNever` inside `integration-tests`? I'd like to avoid the cross-project import especially into the `build` directory. I have admittedly been just copy/pasting `assertNever` around into each project's utils file.

Reply:
Hm, this must have snuck in from an auto-import. I don't need it, going ahead and deleting.